src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -128,7 +129,7 @@ private static class ResourceBundleHolder {
         private static final ResourceBundle RB =
                 ResourceBundle.getBundle(defaultBundleName);
     }
-    private static PrintStream ostream;
+    static PrintStream ostream;
```

Comment:
I don't think we want SecuritySettings accessing LauncherHelper's field like this. I assume printSecuritySetting is just missing a PrintStream parameter so that it can be called to print the security settings to a given stream.